### ISSUE TITLE
bounding doesn't work properly if using offset

### ISSUE THREAD

**Issue by jdeboi** [react-grid-layout/react-draggable#506]
I'm not sure if this is intended behavior, but I might work on a fix. Here's what I'm trying to do:

I want to make my draggable frame responsive, specifically when a user flips a mobile phone between portrait/ landscape, I want the position to readjust. Default position seems to be static once the component is created, so passing a new default prop doesn't change its position. Using position{} works, but I still want the component to be draggable, not controlled. I found a way to get this to work by passing an offset{}, but then the bounding responds to the default position, not the offset+default. 

in short, I think the bounding should be calculated based on default + offset position? Any thoughts are appreciated.
